ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -85,6 +98,76 @@ wan_edge:
           wan_circuit_id: S512
           set_default_route: true
           ip: dhcp
+  # Testing HA and disabling HA
+  node_groups:
+    # SITE_HA_ENABLED
+    - group: Site423
+      cv_pathfinder_region: AVD_Land_West
+      cv_pathfinder_site: Site423
+      uplink_type: p2p-vrfs
+      uplink_switches: [ site-ha-enabled-leaf ]
+      uplink_interfaces: [ Ethernet52 ]
+      nodes:
+        - name: cv-pathfinder-edge2
+          id: 2
+          l3_interfaces:
+            - name: Ethernet1
+              wan_carrier: ATT
+              wan_circuit_id: 423-01
+              set_default_route: true
+              ip: dhcp
+          uplink_switch_interfaces: [Ethernet1]
+        - name: cv-pathfinder-edge3
+          id: 3
+          l3_interfaces:
+            - name: Ethernet2
+              wan_carrier: Colt
+              wan_circuit_id: 10423
+              ip: 172.15.6.6/31
+          uplink_switch_interfaces: [Ethernet2]
+    # SITE_HA_DISABLED
```

Comment:
No need to add new devices here. You could just add the two existing devices to the same node-group but keep every other setting separate.